packages/block-library/src/footnotes/save.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default function Save( { attributes } ) {
+	const { footnotes } = attributes;
+
+	if ( ! footnotes.length ) {
+		return null;
+	}
+
+	return (
+		<ol>
+			{ footnotes.map( ( { id, text } ) =>
+				<li key={ id }>
+					<a
+						id={ id }
+						href={ `#${ id }-anchor` }
+						aria-label={ __( 'Back to content' ) }
```

Comment:
This will be tricky until we have better validation for blocks if a user edits the post with a different locale loaded. Until we get there, how realistically can we a) hardcode a string in English (in Esperanto?) instead of passing it through `__`; or b) can we drop the `aria-label` attribute? More interestingly, is there any accessible standard or convention that indicates the notion of "returning" that is language-independent?

Reply:
Yeah, that's annoying... Emoji? Are they read and translated?

Reply:
This was also a reason against saving the footnotes in the post content, so it can be generated in PHP.